doc/config.rst

```diff
@@ -516,11 +517,7 @@ Stamp Type: LSST_Silicon
 Required keywords to set:
 """""""""""""""""""""""""
 
-    * ``airmass`` = *float_value* (default = 1.2) The airmass to use in FFTs
-    * ``rawseeing`` = *float_value* (default = 0.7) The FWHM seeing at zenith at 500 nm in arc seconds for FFTs.
-    * ``band`` = *str_value* (default = None) The filter band of the observation.
-    * ``det_name`` = *str_value* The name of the detector.
-
+    * ``det_name`` = *str_value* (only required if doing vignetting) The name of the detector.
```

Comment:
I think this is a bit confusing as stands.

det_name is special in that it currently the only(?) variable which is set for use in the imSim codebase with:

base['eval_variables']['sdet_name'] = det_name

in ccd.py. So in out config file we set this to:

det_name: "$det_name"  # This is automatically defined by the LSST_CCD 

I think maybe the documentation here might be more explicit, but I also wonder if we shouldn't make this assignment by default.  Are there cases where someone sets the CCD they want and then overrides this to be another sensor? I see in the test it is not set because we are not doing vingnetting, but that seems a slightly odd thing to have to think about setting.

Finally, I realized we never give an example of what this string might look like (e.g R22_S11). Some people might think that this means e2V or ITL or a dector number.

Reply:
People can use LSST_Silicon without using LSST_CCD.  So if you're using LSST_CCD, you can just use `$det_name`.  But if not, then you need to set it some other way.

Reply:
I added a note in the docs about all this.